src/Service/OS2Forms/CaseSubmissionNormalizers/CaseSubmissionNormalizer.php

```diff
@@ -7,27 +7,17 @@
 use App\Exception\WebformSubmissionException;
 use Doctrine\ORM\EntityManagerInterface;
 
-class CaseSubmissionNormalizer extends AbstractNormalizer implements SubmissionNormalizerInterface
+class CaseSubmissionNormalizer extends AbstractNormalizer
 {
-    /**
-     * Normalizes the properties shared by all case types.
-     */
-    public function normalizeSubmissionData(string $sender, array $submissionData): array
-    {
-        $config = $this->getConfig();
-
-        return $this->handleConfig($sender, $config, $submissionData);
-    }
-
     protected function getConfig(): array
     {
         return [
             'board' => [
-                'os2FormsKey' => 'naevn',
+                'os2forms_key' => 'naevn',
                 'required' => true,
                 'type' => 'entity',
                 'value_callback' => function (string $property, array $spec, array $submissionData, array $normalizedData, EntityManagerInterface $entityManager) {
-                    $board = $entityManager->getRepository(Board::class)->findOneBy(['id' => $submissionData[$spec['os2FormsKey']] ?? null]);
+                    $board = $entityManager->getRepository(Board::class)->findOneBy(['id' => $submissionData[$spec['os2forms_key']] ?? null]);
                     if (null === $board) {
                         throw new WebformSubmissionException('blabla');
```

Comment:
Maybe we could come up with a better exception message, e.g. `sprintf('Cannot get board %s', $boardId)` (assuming `$boardId = $submissionData[$spec['os2forms_key']] ?? null`) or something similar.

Reply:
Oops. Fixed.